components/Downloads/Release/NpmLink.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+
+import { useContext } from 'react';
+import type { FC } from 'react';
+
+import { ReleaseContext } from '@/providers/releaseProvider';
+
+import LinkWithArrow from './LinkWithArrow';
```

Comment:
```suggestion
import LinkWithArrow from '@/components/Downloads/Release/LinkWithArrow';
```

It looks a little strange, maybe we should update the location of the component at some point 🤔

Reply:
True, this should probably be a Common component